--- vapoursynth-R47.2/configure.ac~	2019-08-15 21:01:45.000000000 +0200
+++ vapoursynth-R47.2/configure.ac	2019-10-30 00:23:01.770277122 +0100
@@ -232,7 +232,7 @@
       [
        AM_PATH_PYTHON([3])
 
-       PKG_CHECK_MODULES([PYTHON3], [python-$PYTHON_VERSION])
+       PKG_CHECK_MODULES([PYTHON3], [python-$PYTHON_VERSION-embed])
 
        AC_CONFIG_FILES([pc/vapoursynth-script.pc])
       ]
@@ -290,7 +290,7 @@
 
               AS_IF(
                     [test -z "$PYTHON3_LIBS"],
-                    [PKG_CHECK_MODULES([PYTHON3], [python-$PYTHON_VERSION])]
+                    [PKG_CHECK_MODULES([PYTHON3], [python-$PYTHON_VERSION-embed])]
               )
 
               AS_CASE(
