--- vapoursynth-R45.1/src/core/cpufeatures.c.orig	2018-10-26 23:11:05.000000000 +0200
+++ vapoursynth-R45.1/src/core/cpufeatures.c	2019-04-27 11:23:33.512030568 +0200
@@ -70,6 +70,7 @@
     }
 }
 #elif defined(VS_TARGET_OS_LINUX)
+#if defined(VS_TARGET_CPU_ARM) || defined(VS_TARGET_CPU_POWERPC)
 #include <sys/auxv.h>
 
 void getCPUFeatures(CPUFeatures *cpuFeatures) {
@@ -98,5 +99,11 @@
 #endif
 }
 #else
+void getCPUFeatures(CPUFeatures *cpuFeatures) {
+    memset(cpuFeatures, 0, sizeof(CPUFeatures));
+    cpuFeatures->can_run_vs = 1;
+}
+#endif
+#else
 #warning "Do not know how to get CPU features."
 #endif
--- vapoursynth-R45.1/src/core/cpufeatures.h.orig	2019-04-27 09:40:11.058965473 +0200
+++ vapoursynth-R45.1/src/core/cpufeatures.h	2019-04-27 11:14:27.121657286 +0200
@@ -59,8 +59,6 @@
     char efp_double;
     char dfp;
     char vsx;
-#else
-#warning "No VS_TARGET_CPU_* defined/handled!"
 #endif
 } CPUFeatures;
 
