diff -ur vapoursynth-R48.orig/src/core/cpufeatures.cpp vapoursynth-R48/src/core/cpufeatures.cpp
--- vapoursynth-R48.orig/src/core/cpufeatures.cpp	2019-10-25 08:16:07.000000000 +0200
+++ vapoursynth-R48/src/core/cpufeatures.cpp	2019-12-14 13:39:34.121000000 +0100
@@ -103,6 +103,7 @@
     }
 }
 #elif defined(VS_TARGET_OS_LINUX)
+#if defined(VS_TARGET_CPU_ARM) || defined(VS_TARGET_CPU_POWERPC)
 #include <sys/auxv.h>
 
 static void doGetCPUFeatures(CPUFeatures *cpuFeatures) {
@@ -131,6 +132,12 @@
 #endif
 }
 #else
+void getCPUFeatures(CPUFeatures *cpuFeatures) {
+    memset(cpuFeatures, 0, sizeof(CPUFeatures));
+    cpuFeatures->can_run_vs = 1;
+}
+#endif
+#else
 #warning "Do not know how to get CPU features."
 #endif
 
diff -ur vapoursynth-R48.orig/src/core/cpufeatures.h vapoursynth-R48/src/core/cpufeatures.h
--- vapoursynth-R48.orig/src/core/cpufeatures.h	2019-10-25 08:16:07.000000000 +0200
+++ vapoursynth-R48/src/core/cpufeatures.h	2019-12-14 13:40:09.419000000 +0100
@@ -63,8 +63,6 @@
     char efp_double;
     char dfp;
     char vsx;
-#else
-#warning "No VS_TARGET_CPU_* defined/handled!"
 #endif
 } CPUFeatures;
 
