--- vapoursynth-R50/Makefile.am.orig	2020-07-06 20:53:20.109771817 +0200
+++ vapoursynth-R50/Makefile.am	2020-07-06 20:53:23.166421924 +0200
@@ -85,7 +85,7 @@
 
 
 if X86ASM
-noinst_LTLIBRARIES += libvapoursynth_avx2.la
+noinst_LTLIBRARIES += libvapoursynth_avx2.la libvapoursynth_sse2.la
 
 libvapoursynth_avx2_la_SOURCES = src/core/kernel/x86/generic_avx2.cpp \
 								 src/core/kernel/x86/merge_avx2.c \
@@ -93,13 +93,18 @@
 libvapoursynth_avx2_la_CFLAGS = $(AM_CFLAGS) $(AVX2FLAGS)
 libvapoursynth_avx2_la_CXXFLAGS = $(AM_CXXFLAGS) $(AVX2FLAGS)
 
-libvapoursynth_la_SOURCES += src/core/jitasm.h \
+libvapoursynth_sse2_la_SOURCES = \
 							 src/core/kernel/x86/generic_sse2.cpp \
 							 src/core/kernel/x86/merge_sse2.c \
 							 src/core/kernel/x86/planestats_sse2.c \
 							 src/core/kernel/x86/transpose_sse2.c
 
-libvapoursynth_la_LIBADD += libvapoursynth_avx2.la
+libvapoursynth_sse2_la_CFLAGS = $(AM_CFLAGS) -msse2
+libvapoursynth_sse2_la_CXXFLAGS = $(AM_CFLAGS) -msse2
+
+libvapoursynth_la_SOURCES += src/core/jitasm.h
+
+libvapoursynth_la_LIBADD += libvapoursynth_avx2.la libvapoursynth_sse2.la
 endif # X86ASM
 
 if PYTHONMODULE
